casper/src/test/rholang/RevAddressTest.rho

```diff
@@ -0,0 +1,38 @@
+//scalapackage coop.rchain.rholang.rev
+
+//requires RhoSpec
+new
+  rl(`rho:registry:lookup`), RhoSpecCh,
+  revAddress(`rho:rev:address`),
+  stdlog(`rho:io:stdlog`),
+  setup,
+  test_valid_address, test_invalid_address
+in {
+  rl!(`rho:id:6wnujzcraztjfg941skrtbdkdgbko8nuaqihuhn15s66oz8ro5gwbb`, *RhoSpecCh) |
+  for(@(_, RhoSpec) <- RhoSpecCh) {
+    @RhoSpec!("testSuite", *setup,
+      [
+        ("Validating a valid address returns Nil", *test_valid_address),
+        ("Validating a invalid address returns a non-Nil value", *test_invalid_address),
+      ])
+  } |
+
+  contract setup(returnCh) = {
+    returnCh!([])
+  } |
```

Comment:
would be great if there was a way to skip defining the `setup` contract

Reply:
I'll do that in a future PR